Multisig/UI/SwiftUI/UI Library/Overlays/Snackbar/SnackbarCenter.swift

```diff
@@ -60,9 +60,9 @@ class SnackbarCenter: ObservableObject {
         }
     }
 
-    func show(error: DetailedLocalizedError) {
+    func show(error: Error) {
```

Comment:
@MouazAlzahabi  I don't remember exactly, but I thought you removed it in your previous PR. Should it be like this?

Reply:
but then it breaks our requirement of how error messages should look like. Is there another way to fix that crash?

Reply:
tbh which error is causing the trouble 😞

Reply:
@sche here's the PR https://github.com/gnosis/safe-ios/pull/1469/files

Reply:
In general, I am fine with it if you are fine to remove this hard requirement of displaying messages to users only that satisfy the DetailedLocalizedError format.

Reply:
@DmitryBespalov @sche So let's go with it ?

Reply:
Yes, you can merge it @MouazAlzahabi 
We will adjust it later if needed